about/package-scope.md

```diff
@@ -18,21 +18,257 @@ Currently, the packages that pyOpenSci reviews also need to fall into the
 technical and applied scope of our organization. This scope may expand over time
 as the organization grows.
 
-
-
 ## Is Your Package in Scope For pyOpenSci Review?
 
-pyOpenSci only reviews packages that fall within our specified domain and
-technical scope listed below.
+pyOpenSci reviews packages that fall within a list of specified categories and
+domains. Packages must also meet our technical scope requirements.
 
 If you are unsure whether your package is in scope for review, please
 open a [pre-submission inquiry using a GitHub Issue](https://github.com/pyOpenSci/software-review/issues/new?assignees=&labels=0%2Fpresubmission&template=presubmission-inquiry.md&title=) to get feedback from
 one of our editors. We are happy to look at your package and help you understand
 whether it is in scope or not.
 
-```{include} /appendices/scope.md
+### About the types of packages that we review
+
+pyOpenSci reviews packages that support open reproducible science,
+data processing and and the various stages of managing the
+data lifecycle. Packages submitted to pyOpenSci should fit into one or
+more of the categories below and should be within our technical scope.
+
+```{admonition} Package Use Metrics Are Not a Requirement for Review
+:class: important
+
+When we evaluate whether you package is within our scope, we only consider:
+
+1. how the package is developed and
+2. how the package relates to and supports the broader scientific ecosystem
+
+pyOpenSci does not evaluate how much community use your package has.
+As such, the number of GitHub stars or PyPI / Conda
+downloads is NOT considered as a part of our scope evaluation.
```

Comment:
This is very explicit, but unsure it adds much content.

Reply:
If you're worried about users being confused at this point already, perhaps it's worth clarifying *WHY* you are reviewing in the first place. It's not to establish how useful a package is, but how well it is put together.  This is a subtle change of emphasis on the "we only consider" clause above.

Reply:
ok and now i'm here - 

We review packages with the goal of improving package quality and usability for scientists. 
As such, we review packages across a spectrum of small to large user bases. The popularity of your package is not a consideration in our review process! 

When we evaluate whether you package is within our scope, we only consider:

1. how the package is developed and
2. how the package relates to and supports the broader scientific ecosystem

We welcome young packages that are just entering the scientific Python
ecosystem to apply for review if they are relevant to the science community and
fit into at least one scope category below. We also welcome mature packages with
a growing or established community!